codalab/rest/interpret.py

```diff
@@ -75,7 +75,11 @@ def _interpret_wsearch():
     }
```

Comment:
Can you change this comment from "Returns worksheet items given a search query for worksheets" -- I don't think it returns worksheet items